packages/compass-indexes/src/components/create-index-modal/create-index-modal.tsx

```diff
@@ -70,13 +74,28 @@ function CreateIndexModal({
     undefined
   );
 
+  // @experiment Early Journey Indexes Guidance & Awareness  | Jira Epic: CLOUDP-239367
+  const enableInIndexesGuidanceExp = usePreference('enableIndexesGuidanceExp');
+  const showIndexesGuidanceVariant =
+    usePreference('showIndexesGuidanceVariant') && enableInIndexesGuidanceExp;
+
+  useFireExperimentViewed({
+    testName: TestName.earlyJourneyIndexesGuidance,
+    shouldFire: enableInIndexesGuidanceExp,
+  });
```

Comment:
Shouldn't this fire only if the modal is open only? Otherwise if this is only based on preferences, what's the point of doing it from indexes tab and not, for example, from the mms side when the experiment is set up?